### PR TITLE
Add Intel Gaudi  as ADOPTERS

### PR DESCRIPTION
**Pull request by talcoh2x**
### What this PR does
Before this PR:

After this PR:

Adds [Habana Labs](https://habana.ai/) to adopters.

We are using KubeVirt to run VMs on the Intel® Gaudi® AI accelerator. This allows cost-effective experimentation and provides access to powerful Gaudi.

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
add Intel Gaudi to adopters.
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
...
 | End-user | CoreWeave | 2020 | [link](https://www.coreweave.com) | A Kubernetes native cloud provider with focus on GPUs at scale. KubeVirt allows us to co-locate non-containerizable workloads such as Virtual Desktops next to compute intensive containers executing on bare metal. All orchestrated via the Kubernetes API leveraging the same network policies and persistent volumes for both VM and containerized workloads. |
 | End-user | Genesis Cloud | 2022 | [link](https://genesiscloud.com/) | Genesis Cloud is basing its public cloud offering for instances with GPUs and other accelerators on kubevirt. |
+| End-user | Habana Labs | 2024 | [link](https://habana.ai/) | Habana Labs utilizes KubeVirt and builds upon open-source projects such as Kubernetes and KubeVirt within its core data centers to empower products like  Habana Labs uses KubeVirt and built on open-source projects like [Intel® Gaudi® 3](https://habana.ai/products/gaudi3/) |
```
Comment (fabiand) [inline ADOPTERS.md:17]: The wording looks a little bit redundant. Can you Try to rephrase it a little and maybe include if you are using it to test/build/certify etc your porducts?

**Review by fabiand — COMMENTED** [review]
Great to see you as an adopter!

File: ADOPTERS.md (hunk 1 above)
Comment (talcoh2x) [reply ADOPTERS.md]: Hi @fabiand I cleaned the duplication I had. please check again. we use Kubevirt for AI workloads based on the Intel Gaudi AI accelerator in our internal DC

Comment (fabiand) [reply ADOPTERS.md]: Awesome, thanks, looks good!

FWIW I think understanding the reasoning why running it in VMs vs plain pods is something that many people might be interested in - aka session content this seems to be :)

**Comment by vladikr** [comment]
> Great to see you as an adopter!

+1 Great to see this PR!

**Comment by vladikr** [comment]
@talcoh2x could you please sign your commits with `--signoff` ?

File: ADOPTERS.md (hunk 1 above)
Comment (chen-levkovich) [reply ADOPTERS.md]: Please change "Habana Labs" to "Intel Gaudi" Thanks

**Comment by chen-levkovich** [comment]
Please change "Habana Labs" to "Intel Gaudi" Thanks

**Review by brianmcarey — COMMENTED** [review]
/lgtm

**Pull request merged**